lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SearchDateRangeAdvancedTabbedComponent } from './search-date-range-advanced-tabbed.component';
+import { By } from '@angular/platform-browser';
+import { TranslateModule } from '@ngx-translate/core';
+import { ContentTestingModule } from '../../../testing/content.testing.module';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import {
+    SearchDateRangeAdvancedComponent,
+    SearchFilterTabbedComponent,
+    SearchWidgetSettings
+} from '@alfresco/adf-content-services';
```

Comment:
This import is wrong, you're already inside of content-services

Reply:
Updated imports in test cases